static/src/javascripts/lib/url.js

```diff
@@ -5,6 +5,25 @@ import memoize from 'lodash/functions/memoize';
 
 const supportsPushState = hasPushStateSupport();
 
+/* commercial testing instrument */
+// Returns a map { <bidderName>: true } of bidders
+// according to the pbtest URL parameter
+
+type TestNameMap = { [string]: boolean };
+
+const pbTestNameMap: () => TestNameMap = memoize(
+    (): TestNameMap =>
+        new URLSearchParams(window.location.search)
+            .getAll('pbtest')
+            .reduce((acc, value) => {
+                acc[value] = true;
+                return acc;
+            }, {}),
+    (): string =>
+        // Same implicit parameter as the memoized function
+        window.location.search
+);
+
```

Comment:
Why is this here when it looks completely Prebid-specific?

Reply:
@kelvin-chappell It made the testing almost impossible; the alternative would be to move it into a commercial url module, what do you think?
@katebee might have thoughts on this too